ansible/roles/etcd/tasks/main.yml

```diff
@@ -11,4 +11,4 @@
 
 - name: Call restart etcd2
   include: etcd2_restart.yml
-  when: etcd2_modified == true and etcd_started.changed == false
+  when: etcd2_modified == true and etcd2_started.changed == false
```

Comment:
``etcd_started`` variable is universal. Let's update start_etcd2.yaml and s/etcd2_started/etcd_started/ instead. Later, I would like to see both etcd and etcd2 unified, remove ``etcd2_*`` vars and use ``is_coreos`` (or other is_os vars) instead. Other distributions can introduce ``etcd3`` component as etcd-3.*.* is to be complete rewrite of etcd-2.*.*.